x/storage/keeper/keeper.go

```diff
@@ -637,21 +637,22 @@ func (k Keeper) CreateObject(
 	store.Set(types.GetObjectByIDKey(objectInfo.Id), obz)
 
 	if err = ctx.EventManager().EmitTypedEvents(&types.EventCreateObject{
-		Creator:        operator.String(),
-		Owner:          objectInfo.Owner,
-		BucketName:     bucketInfo.BucketName,
-		ObjectName:     objectInfo.ObjectName,
-		BucketId:       bucketInfo.Id,
-		ObjectId:       objectInfo.Id,
-		CreateAt:       objectInfo.CreateAt,
-		PayloadSize:    objectInfo.PayloadSize,
-		Visibility:     objectInfo.Visibility,
-		PrimarySpId:    sp.Id,
-		ContentType:    objectInfo.ContentType,
-		Status:         objectInfo.ObjectStatus,
-		RedundancyType: objectInfo.RedundancyType,
-		SourceType:     objectInfo.SourceType,
-		Checksums:      objectInfo.Checksums,
+		Creator:             operator.String(),
+		Owner:               objectInfo.Owner,
+		BucketName:          bucketInfo.BucketName,
+		ObjectName:          objectInfo.ObjectName,
+		BucketId:            bucketInfo.Id,
+		ObjectId:            objectInfo.Id,
+		CreateAt:            objectInfo.CreateAt,
+		PayloadSize:         objectInfo.PayloadSize,
+		Visibility:          objectInfo.Visibility,
+		PrimarySpId:         sp.Id,
+		ContentType:         objectInfo.ContentType,
+		Status:              objectInfo.ObjectStatus,
+		RedundancyType:      objectInfo.RedundancyType,
+		SourceType:          objectInfo.SourceType,
+		Checksums:           objectInfo.Checksums,
+		LocalVirtualGroupId: objectInfo.LocalVirtualGroupId,
```

Comment:
There is no lvg binding to the object when created. So it always be 0 here.

Reply:
If the payload size is zero, it will be sealed i think.